### PR TITLE
Add docs for new VS Code and JetBrain releases

### PR DESCRIPTION
**Pull request by MaedahBatool**
This PR adds both missing and new docs for Cody's VS Code and JetBrains plugins. It ships the following:

### VS Code

- Supported LLM models docs with Claude 3 support
- Keybinding support for custom commands
- @-file supporting numbers
- Updating feature parity docs

### JetBrains

- Supported LLM models docs with Claude 3 support
- @-file support for context
- Update pricing docs table 
- Update feature parity docs

### CODE REVIEW

**Comment by enriquegh** [comment]
We mention:
> Your site administrator determines the LLM, and cannot be changed within the editor.

and talk about BYOLLM but we don’t give out the options Cody Enterprise users have when using Cody Gateway. Can we name those?

I see some defaults here but numerating what is possible would help admins:
https://github.com/sourcegraph/sourcegraph/blob/02122d5f87cd845c6f7c4544bd50f4d8213a8ab3/internal/conf/computed.go#L667

https://sourcegraph.com/docs/cody/core-concepts/cody-gateway#configuring-custom-models

File: docs/cody/clients/feature-reference.mdx [hunk 1]
```diff
@@ -17,6 +17,8 @@ Here's a feature parity matrix that compares the capabilities of Cody Clients ac
 | Clear chat history          | ✓           | ✓             | x          | ✓       |
 | LLM Selection               | ✓           | ✓             | x          | x       |
 | Enhanced  Context Selection | ✓           | ✓             | x          | x       |
+| `@-file` support for context | ✓           | ✓             | x          | x       |
+| `@#-symbol` support for context | ✓           | x             | x          | x       |
```
Comment (taylorsperry) [inline docs/cody/clients/feature-reference.mdx:21]: Can we also add here that only VSC supports Ollama right now (in experimental)? I don't think JB gets this out of the box, does it @philipp-spiess ?

**Review by taylorsperry — APPROVED** [review]
One small comment that I'd love @philipp-spiess to confirm, but other than that LGTM!

File: docs/cody/usage-and-pricing.mdx [hunk 2]
```diff
@@ -102,7 +102,7 @@ The following table shows a high-level comparison of the three plans available o
 | **Embeddings (public code)**      | Supported                                                  | Supported                                                                                   | Supported                             |
 | **Keyword Context (local code)**  | Supported                                                  | Supported                                                                                   | Supported                             |
 | **Developer Limitations**         | 1 developer                                                | Up to 50 devs                                                                               | Scalable, consumption-based pricing   |
-| **LLM Support**                   | Anthropic + Starcoder for Chat, Commands, and Autocomplete | Choice of LLMs for Chat (VS Code only), Anthropic + Starcoder for Commands and Autocomplete | Bring Your Own LLM Key (experimental) |
+| **LLM Support**                   | Anthropic + Starcoder for Chat, Commands, and Autocomplete | Choice of LLMs for Chat (VS Code and JetBrains only), Anthropic + Starcoder for Commands and Autocomplete | Bring Your Own LLM Key (experimental) |
```
Comment (philipp-spiess) [inline docs/cody/usage-and-pricing.mdx:105]:
```suggestion
| **LLM Support**                   | Anthropic + StarCoder for Chat, Commands, and Autocomplete | Choice of LLMs for Chat and Commands (VS Code and JetBrains only), StarCoder and Autocomplete | Bring Your Own LLM Key (experimental) |
```

- We don't officially to allow users to change the autocomplete model, it's going to be mostly StarCoder 
- Commands mostly use the model last used in chat and DO NOT use StarCoder
- Capitalization is StarCoder https://huggingface.co/bigcode/starcoder

File: docs/cody/clients/install-vscode.mdx [hunk 3]
```diff
@@ -203,11 +210,19 @@ VS Code will typically notify you when updates are available for installed exten
 
 Cody also works with Cursor, Gitpod, IDX, and other similar VS Code forks. To access VS Code forks like Cursor, select **Sign in with URL and access token** and generate an access token. Next, copy and paste into the allocated field, using `https://sourcegraph.com` as the URL.
 
-## Using Ollama with Cody
+## Supported LLM models
+
+Cody offers a native support for Anthropic and Starcoder for Chat, Commands, and Autocomplete on the **Free** tier. Users on Cody **Pro** have the ability to choose from a list of supported LLM models for Chat, Commands, and Autocomplete. These LLMs are Claude Instant, Claude 2.0, Claude 2.1, Claude 3 (Opus and Sonnet), ChatGPT 3.5 Turbo, ChatGPT 4 Turbo Preview, and Mixtral.
```
Comment (philipp-spiess) [inline docs/cody/clients/install-vscode.mdx:215]: We don't really allow users to change the model for autocomplete

Comment (philipp-spiess) [inline docs/cody/clients/install-vscode.mdx:215]:
```suggestion
Cody offers a native support for Anthropic and Starcoder for Chat, Commands, and Autocomplete on the **Free** tier. Users on Cody **Pro** have the ability to choose from a list of supported LLM models for Chat, Commands, and Autocomplete. These LLMs are Claude Instant, Claude 2.0, Claude 2.1, Claude 3 (Haiku, Opus and Sonnet), ChatGPT 3.5 Turbo, ChatGPT 4 Turbo Preview, and Mixtral.
```

Haiku coming too with the release tomorrow :)

File: docs/cody/clients/install-vscode.mdx [hunk 4]
```diff
@@ -203,11 +210,19 @@ VS Code will typically notify you when updates are available for installed exten
...
+![llm-selection-cody-pro](https://storage.googleapis.com/sourcegraph-assets/blog/cody-vscode-1.8.0/claude3-2.png)
+
+Cody Enterprise users get additional capabilities like BYOLLM (Bring Your Own LLM) key. Your site administrator determines the LLM, and cannot be changed within the editor.
+
+## Supported local Ollama models with Cody
 
 <Callout type="info">Support with Ollama is currently in the Experimental stage and is available for Cody Free and Pro plans.</Callout>
 
-### Cody Autocomplete with Ollama
+### Cody autocomplete with Ollama
```
Comment (philipp-spiess) [inline docs/cody/clients/install-vscode.mdx:225]: I saw some inconsistency here with the capitalization of "autocomplete" (and chat etc.). Do we capitalize it (in which case this edit is maybe a typo) or not (in which case some of the other added paragraphs might have typos).

File: docs/cody/clients/feature-reference.mdx (hunk 1 above)
Comment (philipp-spiess) [reply docs/cody/clients/feature-reference.mdx:21]: Yes that's correct although we could expose the configuration for it if there's interest. I think that's mostly on the JB team if they would think exposing this setting makes sense.

**Review by philipp-spiess — APPROVED** [review]

File: docs/cody/clients/feature-reference.mdx (hunk 1 above)
Comment (MaedahBatool) [reply docs/cody/clients/feature-reference.mdx:21]: Done 👍

File: docs/cody/clients/install-vscode.mdx (hunk 4 above)
Comment (MaedahBatool) [reply docs/cody/clients/install-vscode.mdx:225]: Nice catch

**Pull request merged**